simplified-app-simplye/src/main/resources/META-INF/services/org.nypl.simplified.viewer.audiobook.AudioBookFeedbooksServiceType

```diff
@@ -1 +0,0 @@
-org.nypl.simplified.simplye.FeedbooksConfigurationService
```

Comment:
Looks like this service is unnecessary due to the `MainFeedbooksSecretService` in Core.

Unless I'm misunderstanding something?

Reply:
This looks like a vestigial organ. I think the service just got defined twice, but the app only looks at the first one it encounters anyway.